### PR TITLE
Feature index E

### PR DESCRIPTION
**Pull request by linawolf**
Also moved pictures from 
Documentaion/ApiOverview/ExtensionScanner/Images
And renamed them to CamelCase

### CODE REVIEW

File: Documentation/ApiOverview/ExtensionScanner/Index.rst [hunk 1]
```diff
@@ -37,7 +37,9 @@ This module has been featured on the TYPO3 youtube channel:
    .. youtube:: UdIYDZgBrQU
 
 
-Quick Start
+.. index:: Admin tool; Scan Extension Files
```
Comment (marble) [inline Documentation/ApiOverview/ExtensionScanner/Index.rst:40]: IMHO should be: `Scan extension files` in normal text and the alt text of images. And in the index therefore too, no matter what the guilabel says. Or, if you insist for the index: write `Admin tool; 'Scan Extension Files'`

File: Documentation/ApiOverview/ExtensionScanner/Index.rst [hunk 2]
```diff
@@ -1,34 +1,34 @@
 .. include:: /Includes.rst.txt
-
+.. index:: Extension Scanner
```
Comment (marble) [inline Documentation/ApiOverview/ExtensionScanner/Index.rst:2]: But this is `Extension scanner` in the following headline. I guess that's what should be considered right?!

File: Documentation/ApiOverview/DependencyInjection/Index.rst [hunk 3]
```diff
@@ -93,6 +95,7 @@ be public, will be marked public automatically due to :yaml:`autoconfigure: true
 These classes include Singletons, because they need to be shared with code that uses
 :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance()` and Extbase controllers.
 
+.. index::Dependency injection; Public
```
Comment (marble) [inline Documentation/ApiOverview/DependencyInjection/Index.rst:98]: Add space after index::

**Review by marble — CHANGES_REQUESTED** [review]
1. Please see inline comments
2. Add a space after index:: , one file is affected. It is what we usually have and moreover, I think it is mandatory.

File: Documentation/ApiOverview/DependencyInjection/Index.rst (hunk 3 above)
Comment (linawolf) [reply Documentation/ApiOverview/DependencyInjection/Index.rst]: yeah, copy and paste...

**Review by marble — APPROVED** [review]

**Pull request merged**